src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,82 @@
+package edu.kis.vh.nursery;
+
+/**
+ * Used to create an instance of a counting out rhymer
+ * Class fields are an array of integers called numbers and an integer variable called total
+ * Capacity of the array is set as 12
+ * Default value of the variable total is set as -1
+ * 
+ */
+public class DefaultCountingOutRhymer {
+
+    public static final int INT = -1;
+    public static final int INT1 = 12;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp)?